nekoyume/Assets/_Scripts/Extensions/AvatarStateExtensions.cs

```diff
@@ -82,5 +74,16 @@ public static async Task<List<RuneState>> GetRuneStatesAsync(this AvatarState av
 
             return runeStates;
         }
+
+        public static async Task<CollectionState> GetCollectionStateAsync(this AvatarState avatarState)
```

Comment:
```suggestion
        // never return null
        public static async Task<CollectionState> GetCollectionStateAsync(this AvatarState avatarState)
```
이런식으로 예상 동작을 적어주시면 좋을 것 같습니다.
어떤건 null or Bencodex.Null이 나와야 없는 상태인 경우도 있고 이것처럼 빈 객체를 주기도하고 그런 케이스가 있어서 적어두면 좋을 것 같아요

Reply:
해당 사항은 주석으로 표기하기보다는 nullable로 확인하는게 좋을것같네요.

Reply:
이거 무슨 말인지 이해가 안되는데 자세하게 설명좀 해주실 수 있을까요?

Reply:
AvatarStateExtension.cs에 `#nullable enable` 붙여서 warning 표시되도록 했습니다.